federatedscope/cl/fedgc/client.py

```diff
@@ -0,0 +1,68 @@
+import torch
+import logging
+import copy
+import numpy as np
+
+from federatedscope.core.message import Message
+from federatedscope.core.workers.client import Client
+from federatedscope.core.auxiliaries.utils import merge_dict
+
+logger = logging.getLogger(__name__)
+
+class GlobalContrastFLClient(Client):
+    r"""
+    GlobalContrastFL(Fedgc) Client receive aggregated model weight from server then update local 
+    weight; it also receive global loss from server to train model and update weight locally.
+    """
+    def _register_default_handlers(self):
```

Comment:
is it possible to call base class's method to avoid repeating several of these lines?